assets/scss/components/_togglePill.scss

```diff
@@ -7,14 +7,18 @@
 	@include shadowOnHover();
 	margin: $space-quarter 0;
 	-webkit-tap-highlight-color: rgba(0,0,0,0);
+
+	.radioButtonGroup & {
```

Comment:
I'd prefer to keep things as neatly separated as possible. Instead of this, I'd recommend adding a class of `span--100` or `display--flex` to each TogglePill in the group.

Reply:
:thumbsup: :thumbsup:  :thumbsup: